### ISSUE TITLE
configuring with Rails

### ISSUE THREAD

**Issue by jtmkrueger** [Strech/sidekiq-prometheus-exporter#14]
I'm trying to get this to work with a Rails app, and it's not totally clear how I need to set things up to get the `/metrics` route. I've figured out how to get `/sidekiq/metrics` with those configuration options. Do you have any insight into rails configuration?

**Comment by Strech** [comment]
@jtmkrueger Hola! Thanks for the question. To be clear – you want to mount /metrics from this gem directly into rails routes, correct?

**Comment by jtmkrueger** [comment]
Yea! That's what we've been trying to figure out.

**Comment by Strech** [comment]
@jtmkrueger Hey, thanks for your question. I've updated the readme and added [a section about the Rails](https://github.com/Strech/sidekiq-prometheus-exporter#rails-application)

TL;DR

```ruby
Rails.application.routes.draw do
  # ... omitted ...

  # If you want metrics url to be namespaced replace `at` value with your namespace,
  # otherwise keep it as `/` (maybe at the very bottom of the file)
  require 'sidekiq/prometheus/exporter'
  mount Sidekiq::Prometheus::Exporter, at: '/'
end
```

**Comment by jtmkrueger** [comment]
This is really close! I'll make a PR for what I did differently though

**Comment by Strech** [comment]
@jtmkrueger Thanks for your contribution! The readme was updated 👏
